addons/sourcemod/scripting/vsh/bosses/boss_horsemann.sp

```diff
@@ -136,8 +157,21 @@ methodmap CHorsemann < SaxtonHaleBase
 	
 	public Action OnSoundPlayed(int clients[MAXPLAYERS], int &numClients, char sample[PLATFORM_MAX_PATH], int &channel, float &volume, int &level, int &pitch, int &flags, char soundEntry[PLATFORM_MAX_PATH], int &seed)
 	{
-		if (strncmp(sample, "vo/", 3) == 0)//possibly look replacing into one of HHH sound?
+		if (strncmp(sample, "vo/", 3) == 0)
+		{
+			if (StrContains(sample, "vo/halloween_boss/", false) == 0)
+				return Plugin_Continue;
+			
+			Format(sample, sizeof(sample), "%s", g_strHorsemannVoice[GetRandomInt(0, sizeof(g_strHorsemannVoice) - 1)]);
```

Comment:
`"%s",` not needed, can directly use `g_strHorsemannVoice` without needing optional format parameters